guides/common/modules/ref_system-requirements.adoc

```diff
@@ -109,3 +109,10 @@ endif::[]
 For more information, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/assembly_installing-a-rhel-8-system-with-fips-mode-enabled_security-hardening[Installing a RHEL 8 system with FIPS mode enabled] in the _{RHEL} Security Hardening Guide_.
 For more information about FIPS on {RHEL} 7 systems, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/chap-federal_standards_and_regulations#sec-Enabling-FIPS-Mode[Enabling FIPS Mode] in the _{RHEL} Security Guide_.
 endif::[]
+
+ifdef::satellite[]
+[NOTE]
+====
+{Project} supports DEFAULT and FIPS crypto-policies. The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installation.
```

Comment:
If you want to use it twice, maybe a snippet is also a good idea, similar to `guides/common/modules/snip_firewalld.adoc`. 

With two instances only, I'd say it's your call.

Reply:
Lets keep it this way as there are only two occurrences.